app/lib/methods/audioPlayer.ts

```diff
@@ -0,0 +1,119 @@
+import { AVPlaybackStatus, Audio, InterruptionModeAndroid, InterruptionModeIOS } from 'expo-av';
+import { Sound } from 'expo-av/build/Audio/Sound';
```

Comment:
Is this import correct/needed?
I'm pretty sure you can do Audio.Sound, but it could be a type, so idk